src/test/java/com/codebox/bean/JavaBeanTesterTest.java

```diff
@@ -141,14 +150,23 @@ void test_fullBean() {
     @Test
     void test_fullBeanNullExt() {
         JavaBeanTester.builder(SampleBean.class).checkEquals().loadData().test();
-        JavaBeanTester.builder(SampleValueObject.class).checkEquals().loadData().test();
+        // Will not be able to build a object.
```

Comment:
@pthptht Area of concern, need to check to see why this is excluded now.  This is presumably the small drop in coverage.